index.html

```diff
@@ -295,6 +295,35 @@ <h3 class="lang">Skills</h3>
       <img class="saperator" src="images/Separator Bottom.png" alt="">
   </section>
 
+  <section class="contact_section">
+
+    <h2 class="contact-head">Contact me</h2>
+    <p class="contact_text">If you have an application you are interested in developing, a feature that you need built
+      or a project that
+      needs coding. I’d love to help with it </p>
+
+    <form action="https://formspree.io/f/xleknroo" method="POST">
+      <form id="fs-frm" class="contact_form" name="simple-contact-form" action="https://formspree.io/f/{form_id}"
+        method="post">
+
+        <label for="full-name">
+          <input class="input_name" type="text" name="name" id="full-name" placeholder="Maurice"
+            accept-charset="utf-30">
+        </label>
+        <label>
+          <input class="input_mail" type="email" name="email" placeholder="Olabisi2k11@gmail.com">
+        </label>
+        <label>
+          <textarea class="message_box" name="message" placeholder="write your message here"></textarea>
+        </label>
+        <button class="touch_btn" type="submit">
+         Get in touch
+        </button>
+      </form>
```

Comment:
Kindly make sure the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. 👇 

<img width="945" alt="image" src="https://user-images.githubusercontent.com/73854999/222103155-6d784305-0f56-4286-b128-1a43f2143d12.png">